scripts/zones/Windurst_Walls/npcs/Shantotto.lua

```diff
@@ -19,7 +19,11 @@ function onTrade(player,npc,trade)
     local count = trade:getItemCount()
 
     if wsQuestEvent ~= nil then
-        player:startEvent(wsQuestEvent)
+        if wsQuestEvent == 448 then
+            player:startEvent(wsQuestEvent, nil, nil, 345)
```

Comment:
`dsp.keyItem.ANNALS_OF_TRUTH`

Reply:
@ibm2431  changed, thanks for the review 😸